wp-includes/update.php

```diff
@@ -14,208 +14,166 @@
 }
 
 /**
- * Checks Retraceur version against the newest version.
+ * Look for Retraceur requirements parsing its release note content.
  *
- * The Retraceur version, PHP version, and locale is sent to remote directory provider.
+ * @since 2.0.0 Retraceur fork.
  *
- * @since WP 2.3.0
- * @since 1.0.0 Retraceur fork.
- *
- * @global string $retraceur_version       Used to check against the newest Retraceur version.
- * @global wpdb   $wpdb             WP database abstraction object.
- * @global string $wp_local_package Locale code of the package.
- *
- * @param array $extra_stats Extra statistics.
- * @param bool  $force_check Whether to bypass the transient cache and force a fresh update check.
- *                           Defaults to false, true if $extra_stats is set.
+ * @param string $html HTML content of the release note.
+ * @param string $type 'PHP' or 'MySQL'.
+ * @return string The required version number.
  */
-function wp_version_check( $extra_stats = array(), $force_check = false ) {
-	global $wpdb, $wp_local_package;
+function retraceur_get_requirement( $html, $type = 'PHP' ) {
+	$tags    = new WP_HTML_Tag_Processor( $html );
+	$tag     = 'MySQL' === $type ? 'td' : 'th';
+	$skip    = $type . ' >=';
+	$version = '';
+	$i       = 2;
+
+	while ( $i > 0 && $tags->next_tag( $tag ) ) {
+		$tags->next_token();
+
+		$version = $tags->get_modifiable_text();
+		if ( $skip === $version ) {
+			$version ='';
```

Comment:
```suggestion
			$version = '';
```
Improve code formatting.